andela/ah-kifaru-frontend
# #167827383  Hotfix for footer svg icons

#### What does this PR do?
Fix footer svg icons

#### Description of Task to be completed?
- ensure the footer svg icons are present and not as  javascript components
- ensure dead code is replaced with live code in index.js file in the icons folder

#### How should this be manually tested?
1) Clone the repository and navigate to the `src/assets/icons` folder
2) Observe that there are javascript icon components instead of svg icons
3) There is dead code inside the `index.js` file

#### Any background context you want to provide?
This fix will enable svg icons to be easily imported and used in the project

#### What are the relevant pivotal tracker stories? (if applicable)
#167827383

#### Screenshots
Before this fix:
![image](https://user-images.githubusercontent.com/31288326/62833257-a8a12d80-bc33-11e9-80db-3335565ca425.png)

After this fix:
![image](https://user-images.githubusercontent.com/31288326/62833266-ba82d080-bc33-11e9-8bc8-50588e7be7fe.png)
